Tests/XCUITests/DownloadFilesTests.swift

```diff
@@ -12,30 +12,40 @@ class DownloadFilesTests: BaseTestCase {
     override func tearDown() {
         // The downloaded file has to be removed between tests
         waitForExistence(app.tables["DownloadsTable"])
-
-        let list = app.tables["DownloadsTable"].cells.count
-        if list != 0 {
-            for _ in 0...list-1 {
-                waitForExistence(app.tables["DownloadsTable"].cells.element(boundBy: 0))
-                app.tables["DownloadsTable"].cells.element(boundBy: 0).swipeLeft()
-                waitForExistence(app.tables.cells.buttons["Delete"])
-                app.tables.cells.buttons["Delete"].tap()
+        if processIsTranslatedStr() == m1Rosetta {
+            navigator.nowAt(LibraryPanel_Downloads)
+            navigator.goto(NewTabScreen)
+            navigator.performAction(Action.CloseURLBarOpen)
+            navigator.nowAt(NewTabScreen)
+            navigator.goto(ClearPrivateDataSettings)
+            app.cells.switches["Downloaded Files"].tap()
+            app.tables.cells["ClearPrivateData"].tap()
+            app.alerts.buttons["OK"].tap()
+        } else {
+            let list = app.tables["DownloadsTable"].cells.count
```

Comment:
I am keeping this `swipeLeft()` related code snippet because we still need to ensure that the swipe makes the delete option available.

Reply:
yeah, we have to keep it so that we can run tests on Intel too. Thanks